sec_certs/dataset/dataset.py

```diff
@@ -433,3 +434,10 @@ def compute_related_cves(
         logger.info(
             f"In total, we identified {n_vulnerabilities} vulnerabilities in {n_vulnerable} vulnerable certificates."
         )
+
+    def get_keywords_df(self) -> pd.DataFrame:
```

Comment:
I think I would actually keep the original name, because for CC it makes sense to get a similar dataframe for certification report keywords and do analysis on that. I am not sure how to unify this.

Reply:
I attempted to resolve it with https://github.com/crocs-muni/sec-certs/pull/275/commits/1b150aa58b1ff8effb35649201ad590a41d3cc7e

Reply:
I'm afraid that `getattr()` will be our friend in this PR 😆

Reply:
Nice!